pkg/scheduler/plugins/elasticquota/plugin.go

```diff
@@ -215,8 +216,9 @@ func (g *Plugin) EventsToRegister() []framework.ClusterEventWithHint {
 	// https://git.k8s.io/kubernetes/pkg/scheduler/eventhandlers.go#L403-L410
 	eqGVK := fmt.Sprintf("elasticquotas.v1alpha1.%v", scheduling.GroupName)
 	return []framework.ClusterEventWithHint{
-		{Event: framework.ClusterEvent{Resource: framework.Pod, ActionType: framework.Delete}},
-		{Event: framework.ClusterEvent{Resource: framework.GVK(eqGVK), ActionType: framework.All}},
+		{Event: framework.ClusterEvent{Resource: framework.Pod, ActionType: framework.Delete}, QueueingHintFn: g.isSchedulableAfterPodDeletion},
+		{Event: framework.ClusterEvent{Resource: framework.GVK(eqGVK), ActionType: framework.Add | framework.Delete}},
+		{Event: framework.ClusterEvent{Resource: framework.GVK(eqGVK), ActionType: framework.Update}, QueueingHintFn: g.isSchedulableAfterQuotaChanged},
```

Comment:
Is it better to consider all quota events (not only Update events) with queueing hint so that we can skip events of non-related quota?